rmw_implementation/test/benchmark/benchmark_symbols.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include "performance_test_fixture/performance_test_fixture.hpp"
+
+#include "../../src/functions.cpp"
```

Comment:
```suggestion
#include "../../src/functions.hpp"
```

We should be testing against the real compiler output instead of recompiling it ourself. This will require that you add `prefetch_symbols()` to `functions.hpp`.

Reply:
In the interest of moving this PR along, I addressed this with 0507fc2.